src/components/addComment.js

```diff
@@ -26,26 +29,62 @@ const StyledAddComment = styled.div`
 `;
 
 export default class AddComment extends React.Component {
-  createComment = () => {
-    console.log('create comment for post with id', this.props.postId);
+  state = {
+    description: ''
+  };
+
+  handleChange = e => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  isCommentValid = () => {
+    return this.state.description.length;
   };
 
   render() {
-    const { postId } = this.props;
+    const { postId, user } = this.props;
     return (
-      <StyledAddComment>
-        <img src="http://placehold.it/32x32" alt="me" />
-        <Textarea
-          id={`comment-${postId}`}
-          type="text"
-          placeholder="Add your comment"
-        />
-        <div className="buttons">
-          <Button type="submit" onClick={this.createComment}>
-            Add
-          </Button>
-        </div>
-      </StyledAddComment>
+      <Mutation
+        mutation={CREATE_COMMENT_MUTATION}
+        variables={{
+          description: this.state.description,
+          postId: postId,
+          token: user.groupToken
+        }}
+        refetchQueries={[
```

Comment:
Could consider refetching only the adjusted post - otherwise the graphql advantage is not being used but lets put on the backlog for now